.github/actions/check-pull-request-title/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "check-pull-request-title",
+    "version": "1.0.0",
+    "description": "Simple GitHub action that checks pull request title",
+    "main": "index.js",
+    "scripts": {
+        "build": "ncc build src/action.ts -o build"
+    },
+    "author": "GoodData",
+    "license": "UNLICENSED",
```

Comment:
I'm not sure if `UNLICENSED` is correct

Reply:
That is a very good question :-/ would MIT be better? we do not use any non-MIT stuff here, right?

Reply:
We don't. Let's change it to MIT.